library/std/src/net/addr.rs

```diff
@@ -290,7 +290,22 @@ impl SocketAddrV4 {
     /// ```
     #[stable(feature = "rust1", since = "1.0.0")]
     pub fn ip(&self) -> &Ipv4Addr {
-        unsafe { &*(&self.inner.sin_addr as *const c::in_addr as *const Ipv4Addr) }
+        // FIXME: what about other OSes besides POSIX and Win32 ?
+
+        // SAFETY: Casting `&in_addr` to `&Ipv4Addr` is safe because:
+        //
+        //   * `Ipv4Addr` is `struct { _: [u8; 4]; }` with size = 4; alignment = 1.
+        //
```

Comment:
```suggestion
```

Reply:
I find an empty line between list items is easier to read. But if you and others find the opposite, I will remove it then.